scripts/partition_manager.py

```diff
@@ -222,6 +222,27 @@ def convert_str_to_list(with_str):
             with_str[k].append(v)
 
 
+def resolve_ambiguous_requirements(reqs, unsolved):
+    """
+    Find partitions where the requirements are identical, and therefore ambiguous. For all partitions with identical
+    requirements, introduce requirements so that the partitions have unique placements (sorted by name).
```

Comment:
Lines breaking the 79 char pep rule.

Reply:
do we have 79 char pep rule in ncs?

Reply:
Hm, my pycharm pep didn't pick this up, will fix it.